examples/stable_radical_optimization/submit_combined_with_center.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#SBATCH --account=bpms
+#SBATCH --time=4:00:00
+#SBATCH --job-name=stable_rad_opt
+#SBATCH --mail-type=BEGIN,END,FAIL
+#SBATCH --mail-user=jlaw@nrel.gov
```

Comment:
Forgot I added my email address to the submit script. Could you comment out these two lines? Thanks :)

Reply:
ha! sorry about that if you were getting a bunch of notifications

Reply:
Haha no problem